requirements.txt

```diff
@@ -6,7 +6,7 @@ IntelHex>=1.3
 junit-xml
 pyYAML
 requests
-mbed-ls>=0.2.13
+mbed-ls>=0.2.13,<2.0
```

Comment:
Why do we add a high limit to mbed-ls version ?

Reply:
This is syncronized with icetea. If not then there is risk that icetea and this support different version and cause problems

Reply:
Is this because of https://github.com/ARMmbed/icetea/blob/929dc1da4c2529fff633545c00b677d0970ee0a5/setup.py#L32
It seems we actually also need a version >=1.4.2

Reply:
mbed-ls v0.2.13 is very old - maybe it's better to update this anyway..